src/openMVG/matching/CMakeLists.txt

```diff
@@ -24,14 +24,23 @@ add_library(openMVG_matching
   ${matching_files_header}
   ${matching_files_cpp})
 target_include_directories(openMVG_matching
-  PUBLIC $<BUILD_INTERFACE:${FLANN_INCLUDE_DIRS}>
+  PUBLIC
+    $<BUILD_INTERFACE:${FLANN_INCLUDE_DIRS}>
+    $<INSTALL_INTERFACE:include/>
```

Comment:
We need to stick to `/` or does not put it at the end.
Seems like cmake recommend not to use the last `/`.
See at the end of https://cmake.org/cmake/help/v3.0/command/install.html